src/appleseed/renderer/kernel/lighting/lightpathstream.cpp

```diff
@@ -267,9 +267,19 @@ void LightPathStream::create_path_from_hit_emitter(const size_t emitter_event_in
             reflector_vertex.m_radiance = current_radiance;
             m_vertices.push_back(reflector_vertex);
 
+            // If the hit is an emitter, add the radiance to current
+            if (event.m_type == EventType::HitEmitter)
+            {
+                const auto& event = m_events[event_index];
```

Comment:
Why do you need this declaration? It's already defined before and doesn't change.

Reply:
Yep it's redundant, below too.